packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/steps/Salaires/store/store.ts

```diff
@@ -42,11 +43,14 @@ const createSalairesStore: StoreSlice<
     initFieldSalaries: () => {
       const ancienneteInput = get().ancienneteData.input;
       const contratTravailInput = get().contratTravailData.input;
+      const dateNotification = parse(
+        contratTravailInput.dateArretTravail ??
+          ancienneteInput.dateNotification!
+      );
+      dateNotification.setDate(dateNotification.getDate() + 1);
```

Comment:
Avec date-fns, tu as une méthode pour ajouter des valeurs à une date :
```javascript
// Add the following duration to 1 September 2014, 10:19:50
const result = add(new Date(2014, 8, 1, 10, 19, 50), {
  years: 2,
  months: 9,
  weeks: 1,
  days: 7,
  hours: 5,
  minutes: 9,
  seconds: 30,
})
//=> Thu Jun 15 2017 15:29:20
```

Je trouve cela plus parlant que `+ 1`.

```suggestion
      dateNotification.setDate(add(dateNotification, { days: 1}));
```